river/command/map.zig

```diff
@@ -58,6 +58,12 @@ pub fn map(
     if (mappingExists(mode_mappings, modifiers, keysym, optionals.release)) |current| {
         mode_mappings.items[current].deinit();
         mode_mappings.items[current] = new;
+        // Warn user if they overwrote an existing keybinding using riverctl.
+        out.* = try std.fmt.allocPrint(
+            allocator,
+            "overwrote an existing keybinding: {s} {s}",
+            .{ args[2], args[3] },
```

Comment:
Does hardcoding the indexes 2 and 3 here work properly if e.g. -release is passed?

Reply:
Hmm yeah didn't though of that, it does not, would you prefer that I make it works or just do not print the keys?

Reply:
Yeah nvm the offset is already here, I thought I would need to add it, so wasn't sure if that was worth it, I'll do this